chainer/link_hooks/spectral_normalization.py

```diff
@@ -247,10 +247,12 @@ def _prepare_parameters(self, link, input_variable=None):
                 else:
                     with chainer.using_device(device):
                         _, s, _ = xp.linalg.svd(array)
+                s = fallback._to_chx(s)
             else:
                 _, s, _ = link.xp.linalg.svd(weight_matrix)
+                s0 = chainer.utils.force_array(s[0])
```

Comment:
You should decrease the indent level by one.

```
2019-07-04 17:38:53.134692 STDOUT 2298] | >               link.gamma = variable.Parameter(s0)
-- | --
2019-07-04 17:38:53.134705 STDOUT 2298] | E               UnboundLocalError: Parameterized test failed.
```

Reply:
yeah, I just fixed it :(